pom.xml

```diff
@@ -20,5 +20,7 @@
     <modules>
         <module>utility-belt</module>
         <module>base</module>
+        <module>jmxterm</module>
```

Comment:
Are this two images running tests for base image? Otherwise we should not build them together with base image.

Reply:
these two images don't run tests for the base image or get used by the base image. they are used by the other images for running tests. i could put these two images in another repo and then we would be able to build them in parallel with the base image since they don't have any dependencies.

Reply:
talked offline and decided to leave them here for now